pkgs/development/libraries/libadwaita/default.nix

```diff
@@ -21,7 +21,7 @@
 
 stdenv.mkDerivation rec {
   pname = "libadwaita";
-  version = "1.1.3";
+  version = "1.2.alpha";
```

Comment:
Please move that into a overwrite for bottles if it is really required.

Reply:
So, whats the best approach here, creating a -dev package like @jtojnar suggests and waiting it's merge to proceed with this or just do the override for now?

Reply:
If more packages require the alpha release it should be its own top-level entry.

Reply:
Changed to override.